doc/Language/regexes.pod6

```diff
@@ -1255,6 +1255,47 @@ list of predefined subrules is listed in
 L<S05-regex|https://design.perl6.org/S05.html#Predefined_Subrules> of design
 documents.
 
+=head1 X<Regex Interpolation|regex, Regex Interpolation>
+
+If you want to build a regex using pattern given at the runtime, regex
+interpolation is what you looking for.
+
+There are two ways you can interpolate a string into regex as a pattern,
```

Comment:
And this one becomes (also, there are three ways):

There are three ways you can interpolate a string into a regex as a pattern.That is using C«$pattern», C«<$pattern>», or C«<{$pattern}>».